### ISSUE TITLE
Feature Request: Shared Budgets

### ISSUE THREAD

**Issue by donavanbecker** [rarfell/dimeApp#1]
This would be our go to budget app, if it had shared budgets. So that my wife and I could both see where we are at in our budget for the month.

**Comment by jmraasch** [comment]
Yes!  I agree!  My husband is an airline pilot so he’s away a lot and it’s so hard to track our budget.  I love this app and would love to sync with him.

**Comment by tejasraman** [comment]
This looks to be possible with CloudKit: https://developer.apple.com/documentation/cloudkit/shared_records/sharing_cloudkit_data_with_other_icloud_users

**Comment by ubmit** [comment]
that would be amazing! are there plans to support this in the future?

**Comment by mantasio** [comment]
+1 on this one
